.ci/jobs/elastic+elasticsearch-js+next.yml

```diff
@@ -0,0 +1,11 @@
+---
+- job:
```

Comment:
What is the trigger for this job?

Reply:
It'll inherit the default `github` trigger from `defaults.yml` when JJBB merges the jobs and defaults.

Reply:
Great!
And the `github` trigger means that it will run a build for every commit in the branch?

Reply:
Yep, that's right.

These files will produce the same behavior as the jobs that currently exist on `clients-ci`.